coral/src/app/components/Form.tsx

```diff
@@ -0,0 +1,147 @@
+// ❗️ This implementation mirrors the Aiven core and is kept up to date with it.
```

Comment:
I would drop this mention. I think we should not directly discourage changes in in this file as if they are valid, we should implement them. It is the job for us who have the ability to do change requests to "upstream".

Reply:
Sounds good!